src/booking.js

```diff
@@ -68,15 +68,18 @@ class RecrasBooking {
 
     amountsValid(pack) {
         let hasAtLeastOneProduct = false;
-        this.getLinesNoBookingSize(pack).forEach(line => {
+        for (const line of this.getLinesNoBookingSize(pack)) {
```

Comment:
This line will be compiled to browser-safe javascript, right?

Reply:
Yes, everything is :)